### ISSUE TITLE
Error handling ERC-1155

### ISSUE THREAD

**Issue by rpcerror** [nanmu42/etherscan-api#76]
There is an issue with error handling while using the ERC-1155 function. With err, result or any other error type the code only logs: etherscan server: NOTOK instead of providing an actual error message.

**Comment by nanmu42** [comment]
Yes, collecting result with message into error message would be a improvement.

As a quick walkaround, try enable verbose mode to see the full response.

https://github.com/nanmu42/etherscan-api/blob/74c513586453640037774b7947f605d14ce7b625/client.go#L147

**Comment by nanmu42** [comment]
Is this what you are looking for?

https://docs.etherscan.io/api-endpoints/accounts#get-a-list-of-erc1155-token-transfer-events-by-address

**Comment by nanmu42** [comment]
This is odd.

Does this endpoint need a paid plan or something?
